do.sh

```diff
@@ -25,6 +25,15 @@ if [ -z "${SSH_PORT:-}" ]; then
 fi
 
 
+if [ -z "${ADMIN_USERS:-}" ]; then
+  echo "Admin users (user:pass) [Enter=adminuser:adminpassword]:"
```

Comment:
The proper message, please.

for example: `Enter administrator(s) credentials: i.e: "user:pass": `

Reply:
Done!